openff/evaluator/backends/dask.py

```diff
@@ -823,16 +824,21 @@ def start(self):
         )
 
         if self._resources_per_worker.number_of_gpus > 0:
+            assert self._resources_per_worker.number_of_gpus==1 
```

Comment:
Could this logic be tucked away in an `if self._resources_per_worker.number_of_gpus == 1` block?